doc/classes/CharFXTransform.xml

```diff
@@ -15,25 +19,44 @@
 			<argument index="1" name="default_value" type="Variant">
 			</argument>
 			<description>
+				Returns the value for [code]key[/code] in the [member env] [Dictionary], or [code]default_value[/code] if [code]key[/code] isn't defined in [member env]. If the value's type doesn't match [code]default_value[/code]'s type, this method will return [code]default_value[/code].
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="absolute_index" type="int" setter="set_absolute_index" getter="get_absolute_index" default="0">
+			The index of the current character (starting from 0). Setting this property won't affect drawing.
 		</member>
 		<member name="character" type="int" setter="set_character" getter="get_character" default="0">
+			The Unicode codepoint the character will use. This only affects non-whitespace characters. [method @GDScript.ord] can be useful here. For example, the following will replace all characters with asterisks:
+			[codeblock]
+			# `char_fx` is the CharFXTransform parameter from `_process_custom_fx()`.
+			# See the RichTextEffect documentation for details.
+			char_fx.character = ord("*")
+			[/codeblock]
 		</member>
 		<member name="color" type="Color" setter="set_color" getter="get_color" default="Color( 0, 0, 0, 1 )">
+			The color the character will be drawn with.
 		</member>
 		<member name="elapsed_time" type="float" setter="set_elapsed_time" getter="get_elapsed_time" default="0.0">
+			The time elapsed since the RichTextLabel was added to the scene tree (in seconds). Time stops when the project is paused, unless the [RichTextLabel]'s [member Node.pause_mode] is set to [constant Node.PAUSE_MODE_PROCESS].
```

Comment:
You can use `[RichTextLabel]` for the first occurrence too. I think the current status quo is that we always use `[Class]` hyperlinks whenever we name a class, even if it may generate a bit too many hyperlinks.

(BTW I just thought that in the future, we may want to automate that and simple make all class references hyperlinks, without relying on the documentation writers doing it with the `[Class]` syntax... but that's for another issue :))